bin/deploy

```diff
@@ -11,19 +11,7 @@ set -e
 
 case "${1:-carnival-staging}" in
   carnival-staging)
-    git push --force staging master
-
-    if [ -n "$BUILD" ]; then
-      heroku run --size=PX build
-
-      git checkout -b deploy
-      git commit --allow-empty -m "Trigger deployment"
-      git push --force staging deploy:master
-      git checkout master
-      git branch -D deploy
-
-      heroku ps:scale web=1
-    fi
+    heroku docker:release
```

Comment:
It used to work without `docker-compose` and just run `docker build`, but that was annoying if you were using docker-compose locally for development, because it wouldn't reuse the same image layers. I don't really understand why `docker build .` and `docker-compose build web` don't share layers, but this change was pretty convenient for me.

@pbrisbin Are you saying I need to ad an `app.json`? Does that replace the `Procfile`?